sync-files/action.yaml

```diff
@@ -95,9 +98,17 @@ runs:
             dest_path=$(yq ".dest" /tmp/file-config.yaml)
             replace=$(yq ".replace" /tmp/file-config.yaml)
             delete_orphaned=$(yq ".delete-orphaned" /tmp/file-config.yaml)
+            pre_commands=$(yq ".pre-commands" /tmp/file-config.yaml)
+            post_commands=$(yq ".post-commands" /tmp/file-config.yaml)
+
+            modified_source_path="/tmp/repository/$source_path"
 
-            source_file="/tmp/repository/$source_path"
-            if [ -f "$source_file" ]; then
+            pre_commands=$(echo "$pre_commands" | sed "s|{source}|$modified_source_path|g" | sed "s|{dest}|$dest_path|g")
```

Comment:
imo: I would suggest using `sd` instead of `sed` in this line as well.

Reply:
I think it's okay, but what is the reason? :thinking:

Reply:
> https://github.com/chmln/sd is added because it's more useful than pure sed.

You write that sd is useful. Therefore, I have confirmed it.

Reply:
Yes, but `sed` here is clean enough.
So, can I consider replacing it in the future?

Reply:
I see. I think either is acceptable :+1: